src/views/Home/components/Hero/Hero.scss

```diff
@@ -6,13 +6,36 @@
   overflow: hidden;
   min-height: 100%;
 
+  @mixin flare {
+    position: absolute;
+    z-index: -3;
+    width: 110vw;
+    height: calc(100%);
```

Comment:
What's the difference between `calc(100%)` and `100%`?

Reply:
yea this is a bit of dead code, I'll clean this up